src/System.Management.Automation/commands/utility/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -921,6 +921,17 @@ internal override void Initialize()
                 }
 
                 int columnsOnTheScreen = this.InnerCommand._lo.ColumnNumber;
+                if (columnsOnTheScreen == int.MaxValue)
```

Comment:
Could we please add comments about the special cases?

Reply:
Yes, will add